erts/emulator/nifs/win32/win_prim_file.c

```diff
@@ -491,6 +491,42 @@ posix_errno_t efile_open(const efile_path_t *path, enum efile_modes_t modes,
     }
 }
 
+posix_errno_t efile_from_fd(int fd,
+                            enum efile_modes_t modes,
+                            ErlNifResourceType *nif_type,
+                            efile_data_t **d) {
+    HANDLE handle;
+
+    handle = (HANDLE)_get_osfhandle(fd);
```

Comment:
`_get_osfhandle` invokes the invalid parameter handler when `fd` is invalid, which crashes in debug builds. I'll try to find a workaround that doesn't involve disabling the handler.

Reply:
There doesn't seem to be a straightforward way to do this, so I think it's best to disable the handler temporarily just for this call.

https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/set-invalid-parameter-handler-set-thread-local-invalid-parameter-handler?view=msvc-160

Reply:
Thanks, I will disable the handler temporary.

Reply:
Fixed: 62a29aa